desktop/libs/aws/src/aws/conf.py

```diff
@@ -288,10 +288,7 @@ def is_enabled():
       has_iam_metadata() or \
       conf_idbroker.is_idbroker_enabled('s3a') or \
       IS_SELF_SIGNING_ENABLED.get() or \
-      (
-        RAZ.IS_ENABLED.get() and bool(get_raz_s3_default_bucket()) and \
-        'default' in list(AWS_ACCOUNTS.keys()) and AWS_ACCOUNTS['default'].get_raw()
-      )
+      ('default' in list(AWS_ACCOUNTS.keys()) and AWS_ACCOUNTS['default'].get_raw())
```

Comment:
@romainr  here do we need to remove these 2 also or no? As these are getting checked in Line 287 but are with one more value (Access key id)

Reply:
Same

Reply:
My gut feeling too is that we could have an extra check if one of the prop is set, as line 291 might be always true
`AWS_ACCOUNTS['default'].HOST.get()
and we won't turn this on when ADLS is on

Reply:
Added raz.is_enabled back but removed the bucket check like before.